tango_simlib/tango_sim_generator.py

```diff
@@ -192,12 +183,14 @@ def attr(tango_device_instance, new_val):
         # Add the read method and the attribute to the class object
         setattr(tango_device_class, read_meth.__name__, read_meth)
         setattr(tango_device_class, attr.__name__, attr)
-        MODULE_LOGGER.info("Adding static attribute {} to the device.".format(attr_name))
+        MODULE_LOGGER.info(
+            "Adding static attribute {} to the device.".format(attr_name)
+        )
```

Comment:
For consistency, See line https://github.com/ska-sa/tango-simlib/blob/user/amakhaba/MT-738/Applied-futurize-stage-2/tango_simlib/tango_sim_generator.py#L173+L175